vision/product_search/add_product_to_product_set.go

```diff
@@ -0,0 +1,40 @@
+// Copyright 2018 Google Inc. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+package productsearch
```

Comment:
Please add a package comment.
```suggestion
// Package productsearch contains samples for Google Cloud Product Search.
package productsearch
```